<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>262</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_filePath">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>178</y>
     <width>128</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>178</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_recv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>27</y>
     <width>161</width>
     <height>76</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_send">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>125</y>
     <width>161</width>
     <height>76</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>接收区：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>108</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>发送区：</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>87</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="text">
     <string>本机UDP端口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_UdpPort"/>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>143</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="text">
     <string>循环间隔时间：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_delayTime"/>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>115</y>
     <width>193</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="text">
     <string>本机TCP端口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_TcpPort"/>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>58</y>
     <width>193</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="text">
     <string>目标端口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_targetPort"/>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>30</y>
     <width>193</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>目 标 IP：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_targetIp">
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnTry">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>207</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>测试通信</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>207</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
